zipkin-server/src/test/java/zipkin2/server/internal/ITZipkinMetricsHealth.java

```diff
@@ -45,7 +45,10 @@
 @SpringBootTest(
   classes = ZipkinServer.class,
   webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
-  properties = "spring.config.name=zipkin-server"
+  properties = {
+    "spring.config.name=zipkin-server",
+    "spring.main.web-application-type=none"
```

Comment:
this might be redundant but this test currently does starting. making that not the case would be progress!